abci/tests/test_app/app.go

```diff
@@ -48,24 +48,6 @@ func deliverTx(client abcicli.Client, txBytes []byte, codeExp uint32, dataExp []
 	}
 }
 
-/*func checkTx(client abcicli.Client, txBytes []byte, codeExp uint32, dataExp []byte) {
```

Comment:
dam, whats your take on the possibility of removing counter app and just supporting kvstore? It seems counter does not provide anything extra?

Reply:
fine by me